modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -47,6 +47,19 @@ $ oc patch argocd openshift-gitops \
 -n openshift-gitops --type=merge \ 
 --patch-file out/argocd/deployment/argocd-openshift-gitops-patch.json
 ----
++
+[NOTE]
+====
+For a disconnected environment, you must patch the `out/argocd/deployment/argocd-openshift-gitops-patch.json` file with the `ztp-site-generator` image mirrored in your local registry by running the following command:
+[source,terminal]
+----
+$ oc patch argocd openshift-gitops -n openshift-gitops --type='json' \
+-p='[{"op": "replace", "path": "/spec/repo/initContainers/0/image", \
+"value": "${LOCAL_REGISTRY}/${ZTP_SITE_GENERATOR_IMAGE_REF}"}]'
```

Comment:
`${LOCAL_REGISTRY}` seems to suggest bash substring replacement but we don't set this up earlier in the procedure, can we simplify with `<local_registry>/<ztp_site_generate_image_ref>`?

Also, is the standard to lowercase user-replaced values? https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
makes sense, Thanks